tests/e2e/support/mutations.js

```diff
@@ -75,6 +75,38 @@ export function createDiscountCode(client, cartDiscountDraft, code) {
     })).then(response => response.data.createDiscountCode);
 }
 
+export function createProduct(client, draft) {
+  return client.mutate({
+    mutation: gql`
+      mutation createNewProduct($draft: ProductDraft!){
+        createProduct (draft: $draft) {
+        id
+        version
+      }
+  }`,
+    variables: {
+      draft,
+    },
+  }).then(response => response.data.createProduct);
+}
+
+export function updateProduct(client, product) {
```

Comment:
This `product` is not exactly a product, right?
It's more the product version, the product id and the actions.
What you can do to avoid giving it a name (which would be difficult to find), is to destructure the object: `{ id, version, actions }`.

Reply:
Alex worked on this file @plaskevich